docker-local/auth-service/spec_files/rules_response/default.json

```diff
@@ -0,0 +1,180 @@
+[
```

Comment:
this file shouldn't exist anymore, artitact commit?

Reply:
Hmm maybe it was a merge issue. I'll clean it up in the next pr - I just want to get this merged so I can talk about it at the convening.